presto-docs/src/main/sphinx/release/release-0.256.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.256
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add support for lambda expressions in ``PREPARE`` statements.
+* Add configuration property ``experimental.distinct-aggregation-spill-enabled`` and session property ``distinct_aggregation_spill_enabled`` to disable spilling for distinct aggregations.
+* Add configuration property ``experimental.order-by-aggregation-spill-enabled`` and session property ``order_by_aggregation_spill_enabled`` to disable spilling for order by aggregations.
+* Add double quotes around the column domain values in the text query plan. Literal double quotes in values will be escaped with a backslash (``ab"c`` -> ``ab\"c``).
+* Add function :func:`array_frequency`.
+* Add support for :func:`zip` to take upto 7 arguments.
+* Add support for ``REFRESH MATERIALIZED VIEW``.
```

Comment:
is there a link to documentation? all new language features should be documented.  I also don't see documentation for ``CREATE MATERIALIZED VIEW``

Reply:
That is a good point. I checked the PR and there is no documentation in the PR :( https://github.com/prestodb/presto/pull/15996/files 

What would you recommend in this situation ? 

Also @highker - your thoughts?

Reply:
Is the feature complete?  I'm not sure the feature is that usable without documentation.  Maybe leave out the release note and @gggrace14 or @highker can add documentation for `CREATE MATERIALIZED VIEW` and `REFRESH MATERIALIZED VIEW` and we can add a release note then.

Reply:
Yes, the feature is complete with CREATE, REFRESH & SELECT from MATERIALIZED VIEW. We would love to add documentation and release note.

We are allowed to add release note afterwards in the repo, is it correct? I will sync up w/ @highker offline on how to add documentation and release note. Sorry, first time to do it.

Reply:
I will remove the release note from this release. @gggrace14 feel free to introduce another release note with link to the doucmentation in the release note